backend/main.py

```
@@ -1586,6 +1586,39 @@ async def download_in_background():
     return {"message": f"Model {request.model_name} download started"}
 
 
+@app.post("/models/download/cancel")
+async def cancel_model_download(request: models.ModelDownloadRequest):
+    """Cancel or dismiss an errored/stale download task."""
+    task_manager = get_task_manager()
+    progress_manager = get_progress_manager()
+
+    removed = task_manager.cancel_download(request.model_name)
```

Ruff (0.15.4)

[error] Local variable `removed` is assigned to but never used

Remove assignment to unused variable `removed`

(F841)